dask_geopandas/geohash.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pandas as pd
+from pygeohash import encode
+
+
+def _geohash(gdf, precision):
+
+    """
+    Calculate geohash for the mid points of each geometry
+    int coordinates
+
+    Parameters
+    ----------
+    gdf : GeoDataFrame
+    precision : int
+        precision of the Geohash
+
+    Returns
+    ---------
+    Pandas Series containing geohash
+    """
+
+    # Calculate bounds
+    bounds = gdf.bounds.to_numpy()
+    # Calculate mid points based on bounds
+    x_mids, y_mids = _calculate_mid_points(bounds)
+    # Vectorize geohash for fast speed up
```

Comment:
Just a note, since this comment may suggest otherwise: `np.vectorize` does not actually provide any performance improvements over looping (as per [documentation](https://numpy.org/doc/stable/reference/generated/numpy.vectorize.html)).

Reply:
That is correct. @tastatham I'd say keep the implementation but change the comment.

Reply:
updated in b8f6e38